app/interfaces/api/v2/case_workers/claim.rb

```diff
@@ -82,7 +82,12 @@ def claims_scope
             end
 
             def claims
-              claims_scope.sort(params.sorting, params.direction).page(params.page).per(params.limit)
+              claims_scope
+                .includes(:external_user, :case_type, :injection_attempts,
```

Comment:
includes most of the associations that are used by the API claim presenter to avoid `N+1` queries.